<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AgenceBancaire</class>
 <widget class="QWidget" name="AgenceBancaire">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AgenceBancaire</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>431</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Connexion BDD</string>
    </attribute>
    <widget class="QLineEdit" name="lineEditAdresseIp">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>80</y>
       <width>411</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>172.18.58.7</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Adresse IP :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mot de passe :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom de la base de données :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>58</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Login :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditNomDeLaBase">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>140</y>
       <width>411</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>ciel2_eledig</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditLogin">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>190</y>
       <width>411</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>ciel2023_eledig</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMotDePasse">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>240</y>
       <width>411</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>V8kY4zV2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonConnexion">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>300</y>
       <width>421</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Connexion</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Ajout Client</string>
    </attribute>
    <widget class="QComboBox" name="comboBoxAgence">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>290</y>
       <width>451</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>58</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>58</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Prénom :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Numéro de compte :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>58</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Agence :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>58</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Solde :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>58</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Ville :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditNom">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>441</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSolde">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>180</y>
       <width>441</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditNumCompte">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>130</y>
       <width>441</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPrenom">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>441</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditVille">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>240</y>
       <width>451</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAjouter">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>350</y>
       <width>481</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Consultation Clients</string>
    </attribute>
    <widget class="QLineEdit" name="lineEditNombreDeClientDansLagence">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>20</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="tableViewComptes">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>601</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre de clients dans l'agence :</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxAgences2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
